libs/core/src/lib/wizard/wizard-progress-bar/wizard-progress-bar.directive.ts

```diff
@@ -0,0 +1,22 @@
+import { Directive, Input } from '@angular/core';
+
+export type WizardSize = 'sm' | 'md' | 'lg' | 'xl';
```

Comment:
why not use enum here?
```typescript
import { Directive, Input } from '@angular/core';

export const enum WizardSize =  {
  SM = 'sm',
  MD = 'md',
  LG = 'lg',
  XL = 'xl'
}

@Directive({
    // tslint:disable-next-line:directive-selector
    selector: '[fd-wizard-progress-bar]',
    host: {
        class: 'fd-wizard__progress-bar',
        '[class.fd-wizard__progress-bar--s]': 'size === WizardSize.SM',
        '[class.fd-wizard__progress-bar--m]': 'size === WizardSize.MD',
        '[class.fd-wizard__progress-bar--l]': 'size === WizardSize.LG',
        '[class.fd-wizard__progress-bar--xl]': 'size === WizardSize.XL'
    }
})
export class WizardProgressBarDirective {
    /**
     * Size of the wizard progress bar.
     */
    @Input()
    size: WizardSize;
}
```

Reply:
What's the advantage?

Reply:
one place of truth
as WizardSize for export, we give a user ability to select from predefined sizes and if we need to change the existing WizardSize values we need to only change in one place

Reply:
Per my other comment: 

"If we wanted to use an enum here then the application developer would have to import the StepType enum rather than just passing strings as inputs and I'd prefer to leave that option as it's what we're doing with every other component (see various fdType examples). I will use constants"